doozer/doozerlib/metadata.py

```diff
@@ -463,6 +463,7 @@ def latest_build_list(pattern_suffix):
                 # Include * after pattern_suffix to tolerate other release components that might be introduced later.
                 # Also include a .el<version> suffix to match the new build pattern
                 rhel_pattern = f'{pattern_prefix}{extra_pattern}{pattern_suffix}.{f"el{el_target}" if el_target else ""}*'
+                assert 'None' not in rhel_pattern
```

Comment:
Not sure I am happy about this assert, as a non-payload weirdness could prevent us from creating 4.16 nightlies. I'd lgtm without this assert.

Reply:
If we're using a pattern like `openshift-base-nodejs-container-v4.16.*.stream.None` we're definitely doing something wrong, and we won't find any builds. This assert is there to ensure we did not screw up in translating a missing el target into the pattern. If we did, we better know asap, and failing the job is the fastest way

Reply:
makes sense, thanks!